ui/src/components/sermons/OnlinePageTabs.js

```diff
@@ -0,0 +1,175 @@
+import {
+  Button,
+  Link,
+  Tabs,
+  Tab,
+  TabList,
+  TabPanels,
+  TabPanel,
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+  AccordionIcon,
+  AspectRatio,
+} from '@chakra-ui/react';
+import { ChevronLeftIcon, RepeatIcon } from '@chakra-ui/icons';
+import { useState } from 'react';
+
+const OnlinePageTabs = ({ noteId, sermonNotes, refreshCallback }) => {
```

Comment:
[NO-ACTION] related to my comment about `OnlinePageTabs.js` handling refresh sermon notes by itself, just needs 
```
const [noteId. setNoteId] = useState(0);
```